### ISSUE TITLE
[BUG] - SelectWrapper with theme=dark is unreadable in native mode without actual dark system UI

### ISSUE THREAD

**Issue by Rouby** [porsche-design-system/porsche-design-system#3501]
### Contact Details

jonathan.burke@codecentric.de

### JS Framework

React

### Porsche Design System version

3.13.0

### On which device(s) you can see the bug?

Windows 11 with touch capabilities

### Reproduction Example

https://stackblitz.com/edit/gdw6ez-g6vrws?file=index.html

### Steps to reproduce the behavior

1. Go to StackBlitz
2. Be on a touch device (or simulate one) on Windows
3. Click on the select wrapper
4. See wrongly styled elements in native select

![image](https://github.com/user-attachments/assets/8eb04fe2-7ac7-484e-8b4a-0989ba12166c)

### Expected behavior

The text should be readable in native select.

### Notes
- Check if `p-select` and `p-multi-select` are affected too
- Instead of using `background: transparent` for styling it might be an option to use something like `rgba(255,255,255,0.01) / rgba(0,0,0,0.01)` to achieve transparency but tell nested native elements like option list which color mode to use. Transparency is needed for visual reasons since form elements can be placed on several surface colors and on images.

**Comment by Rouby** [comment]
Seems like it is linked to the transparent background set here:

https://github.com/porsche-design-system/porsche-design-system/blob/main/packages/components/src/styles/form-styles.ts#L53

**Comment by henrijoss** [comment]
On Windows the options always inherit the `background-color` from the `select`. As far as i can tell the problems only exists for Windows with the `native` prop / touch device and isn't an issue for other mobile devices running iOS/Android. 

One solution is to style the options background depending on the theme:
```
select > option {
   background-color: theme === 'dark' ? themeDarkBackgroundBase : themeLightBackgroundBase;
}
```

Since this style needs to be applied on the slotted options we cannot style it directly from the web component since `::slotted(select > option)` does not work. We could add the style via a constructible stylesheet, which would need to be replaced whenever the theme changes.

**Comment by henrijoss** [comment]
![Image](https://github.com/user-attachments/assets/2b489821-8c11-4a37-9bf9-edf3f20a24b6)
![Image](https://github.com/user-attachments/assets/fa3e6c8d-a757-496a-bd70-80b23add9b68)

Fixed by adding some select > option styles via constructible stylesheet regardless of theme. Styles will be ignored in most iOS/Android Browsers which don't have this issue.
